Build failure due to `relaton-bib` parse failure of some "date"

The builds are failing after compiling all documents successfully, at a place within `relaton-bib`:

https://github.com/CalConnect/cc-admin-documents/actions/runs/12946594193

```
[info]: Building collection file: _site/documents.xml ...
bundler: failed to load command: metanorma (/usr/local/bundle/bin/metanorma)
/usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib.rb:58:in `strptime': invalid date (Date::Error)
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib.rb:58:in `format_date'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib.rb:39:in `parse_date'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/bibliographic_date.rb:25:in `initialize'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:363:in `new'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:363:in `block in fetch_dates'
	from /usr/local/bundle/gems/nokogiri-1.16.8-x86_64-linux/lib/nokogiri/xml/node_set.rb:235:in `block in each'
	from /usr/local/bundle/gems/nokogiri-1.16.8-x86_64-linux/lib/nokogiri/xml/node_set.rb:234:in `upto'
	from /usr/local/bundle/gems/nokogiri-1.16.8-x86_64-linux/lib/nokogiri/xml/node_set.rb:234:in `each'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:360:in `each_with_object'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:360:in `fetch_dates'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:45:in `item_data'
	from /usr/local/bundle/gems/relaton-bib-1.20.2/lib/relaton_bib/xml_parser.rb:18:in `from_xml'
	from /usr/local/bundle/gems/relaton-calconnect-1.20.0/lib/relaton_calconnect/processor.rb:38:in `from_xml'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli.rb:81:in `parse_xml'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/bibdata.rb:41:in `from_xml'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:196:in `bibdata_instance'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:150:in `block in concatenate_files'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:142:in `each'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:142:in `reduce'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:142:in `concatenate_files'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:86:in `bibcollection'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:157:in `concatenate_and_write_to_files'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:19:in `concatenate'
	from /usr/local/bundle/gems/relaton-cli-1.20.0/lib/relaton/cli/relaton_file.rb:58:in `concatenate'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/site_generator.rb:76:in `build_collection_file'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/site_generator.rb:39:in `block in generate'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/site_generator.rb:38:in `chdir'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/site_generator.rb:38:in `generate'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/site_generator.rb:26:in `generate'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli/commands/site.rb:33:in `generate'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/command.rb:28:in `run'
	from /usr/local/bundle/gems/thor-hollaback-0.2.1/lib/thor/hollaback.rb:68:in `run'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor.rb:538:in `dispatch'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/invocation.rb:116:in `invoke'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor.rb:338:in `block in subcommand'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/command.rb:28:in `run'
	from /usr/local/bundle/gems/thor-hollaback-0.2.1/lib/thor/hollaback.rb:68:in `run'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor.rb:538:in `dispatch'
	from /usr/local/bundle/gems/thor-1.3.2/lib/thor/base.rb:584:in `start'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/lib/metanorma/cli.rb:34:in `start'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/exe/metanorma:25:in `block in <top (required)>'
	from /usr/local/bundle/gems/metanorma-cli-1.11.5/exe/metanorma:43:in `<top (required)>'
	from /usr/local/bundle/bin/metanorma:25:in `load'
	from /usr/local/bundle/bin/metanorma:25:in `<top (required)>'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli/exec.rb:59:in `load'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli/exec.rb:59:in `kernel_load'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli/exec.rb:23:in `run'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli.rb:452:in `exec'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/vendor/thor/lib/thor.rb:538:in `dispatch'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli.rb:35:in `dispatch'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/cli.rb:29:in `start'
	from /usr/local/bundle/gems/bundler-2.6.3/exe/bundle:28:in `block in <top (required)>'
	from /usr/local/bundle/gems/bundler-2.6.3/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
	from /usr/local/bundle/gems/bundler-2.6.3/exe/bundle:20:in `<top (required)>'
	from /usr/local/bundle/bin/bundle:25:in `load'
	from /usr/local/bundle/bin/bundle:25:in `<main>'
Error: Process completed with exit code 1.
```

@andrew2net can you please help check? Thanks!

--- a/metanorma.yml
+++ b/metanorma.yml
@@ -67,7 +67,7 @@ metanorma:
     - sources/cc-0810-report-roundtable-12/document.adoc
     - sources/cc-0811-report-roundtable-13/document.adoc
 
-    - sources/cc-0812-pres-meet-cc/collection.yml
+    # - sources/cc-0812-pres-meet-cc/collection.yml
     - sources/cc-0812-pres-meet-cc/pres-intro-01.adoc
     - sources/cc-0812-pres-meet-cc/pres-intro-02.adoc
     - sources/cc-0812-pres-meet-cc/pres-iop-testing.adoc

--- a/sources/cc-0514-dst-adv-notice/document.adoc
+++ b/sources/cc-0514-dst-adv-notice/document.adoc
@@ -6,7 +6,7 @@
 :edition: 1
 :status: published
 :revdate: 2005-06-29
-:published-date: 2005-16-29
+:published-date: 2005-06-29
 :technical-committee: DST ADHOC
 :mn-document-class: cc
 :mn-output-extensions: xml,html,pdf,rxl

--- a/sources/cc-0812-pres-meet-cc/collection.yml
+++ b/sources/cc-0812-pres-meet-cc/collection.yml
@@ -1,0 +1,45 @@
+---
+directives:
+- documents-inline
+
+bibdata:
+  title:
+  - language: en
+    content: CalConnect 0812
+  type: standard
+
+  docid:
+    type: cc
+    id: CC 0812
+  edition: 12
+  copyright:
+    owner:
+      name: CalConnect
+      abbreviation: CalConnect
+    from: '2008'
+format:
+  - html
+  - xml
+  - pdf
+  - presentation
+
+manifest:
+  level: collection
+  title: CalConnect 0812
+
+  docref:
+
+    - file: pres-intro-01.adoc
+    - file: pres-intro-02.adoc
+    - file: pres-iop-testing.adoc
+    - file: pres-ischedule.adoc
+    - file: pres-tc-mobile.adoc
+    - file: pres-bedework.adoc
+    - file: pres-stockholm.adoc
+
+prefatory-content:
+|
+
+
+final-content:
+|

--- a/sources/cc-0812-pres-meet-cc/pres-bedework.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-bedework.adoc
@@ -1,5 +1,5 @@
 = CalConnect and Open Source
-:docnumber: 0812
+:docnumber: 0812-6
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-intro-01.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-intro-01.adoc
@@ -1,5 +1,5 @@
 = An Introduction to The Calendaring and Scheduling Consortium
-:docnumber: 0812
+:docnumber: 0812-1
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-intro-02.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-intro-02.adoc
@@ -1,5 +1,5 @@
 = Introduction to CalConnect: The Calendaring and Scheduling Consortium
-:docnumber: 0812
+:docnumber: 0812-2
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-iop-testing.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-iop-testing.adoc
@@ -1,5 +1,5 @@
 = Interoperability Testing and Standards Development
-:docnumber: 0812
+:docnumber: 0812-3
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-ischedule.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-ischedule.adoc
@@ -1,5 +1,5 @@
 = iSchedule TC
-:docnumber: 0812
+:docnumber: 0812-4
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-stockholm.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-stockholm.adoc
@@ -1,5 +1,5 @@
 = Stockholm University -- Membership in CalConnect
-:docnumber: 0812
+:docnumber: 0812-7
 :copyright-year: 2008
 :language: en
 :doctype: administrative

--- a/sources/cc-0812-pres-meet-cc/pres-tc-mobile.adoc
+++ b/sources/cc-0812-pres-meet-cc/pres-tc-mobile.adoc
@@ -1,5 +1,5 @@
 = CalConnect Technical Committees: TC Mobile
-:docnumber: 0812
+:docnumber: 0812-5
 :copyright-year: 2008
 :language: en
 :doctype: administrative

